doc/sphinx/start_install.rst

```diff
@@ -105,8 +105,18 @@ You can put the observational data and model output in different locations, e.g.
 
 .. _ref-conda-install:
 
-Installing dependencies via the conda package manager
------------------------------------------------------
+Installing dependencies
+-----------------------
+
+Installing XQuartz on MacOS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you're installing on a MacOS system, you will need to install `XQuartz <https://www.xquartz.org/>`__ if it's not already present. 
```

Comment:
Maybe instruct someone on how to check if the have xQuartz installed?  e.g. look for `/Applications/Utilities/XQuartz.app`?

Reply:
Added this in the commit below, although one could argue it's not strictly necessary: the installer is a small download, and since it uses the MacOS graphical installer it handles preexisting installations, etc.